liiatools/datasets/social_work_workforce/SWFtools/analysis/seniority.py

```diff
@@ -91,122 +131,127 @@ def seniority_forecast_04():
         # Newham
-        dfSen.loc[dfSen['LEAName'] == 'Newham', str(countYearNext)] = ((dfSen[str(
-            countYearBefore)] / p_df.loc[3, str(countYearBefore)])*p_df.loc[3, str(countYearNext)])
+        dfSen.loc[dfSen["LEAName"] == "Newham", str(countYearNext)] = (
+            dfSen[str(countYearBefore)] / p_df.loc[3, str(countYearBefore)]
+        ) * p_df.loc[3, str(countYearNext)]
         # Tower Hamlets
-        dfSen.loc[dfSen['LEAName'] == 'Tower Hamlets', str(countYearNext)] = ((dfSen[str(
-            countYearBefore)] / p_df.loc[4, str(countYearBefore)])*p_df.loc[4, str(countYearNext)])
+        dfSen.loc[dfSen["LEAName"] == "Tower Hamlets", str(countYearNext)] = (
+            dfSen[str(countYearBefore)] / p_df.loc[4, str(countYearBefore)]
+        ) * p_df.loc[4, str(countYearNext)]
         # Waltham Forest
-        dfSen.loc[dfSen['LEAName'] == 'Waltham Forest', str(countYearNext)] = (
-            (dfSen[str(countYearBefore)] / p_df.loc[5, str(countYearBefore)])*p_df.loc[5, str(countYearNext)])
+        dfSen.loc[dfSen["LEAName"] == "Waltham Forest", str(countYearNext)] = (
+            dfSen[str(countYearBefore)] / p_df.loc[5, str(countYearBefore)]
+        ) * p_df.loc[5, str(countYearNext)]
 
-    dfSen['2020'] = dfSen['2020'].round(3)
-    dfSen['2021'] = dfSen['2021'].round(3)
-    dfSen['2022'] = dfSen['2022'].round(3)
-    dfSen['2023'] = dfSen['2023'].round(3)
-    dfSen['2024'] = dfSen['2024'].round(3)
-    dfSen['2025'] = dfSen['2025'].round(3)
-
-    dfSen = dfSen.drop(['YearCensus'], axis=1)
+    dfSen["2020"] = dfSen["2020"].round(3)
+    dfSen["2021"] = dfSen["2021"].round(3)
+    dfSen["2022"] = dfSen["2022"].round(3)
+    dfSen["2023"] = dfSen["2023"].round(3)
+    dfSen["2024"] = dfSen["2024"].round(3)
+    dfSen["2025"] = dfSen["2025"].round(3)
 
+    dfSen = dfSen.drop(["YearCensus"], axis=1)
 
     # ===== Save and export file ===== #
-    fileOutN = 'seniority_forecast_04_clean.xlsx'
+    fileOutN = "seniority_forecast_04_clean.xlsx"
     requestPath = work_path.request
     fileOut = os.path.join(requestPath, fileOutN)
     dfSen.to_excel(fileOut, index=False, merge_cells=False)
-    
+
 
 def seniority_forecast_5c():
 
     # ===== Read file ===== #
-    file = 'Seniority.csv'
+    file = "Seniority.csv"
     path = work_path.flatfile_folder
     requestPath = work_path.request
     pathFile = os.path.join(requestPath, file)
     dfSen = pd.read_csv(pathFile)
 
-    dfSen['OrgRoleName'] = dfSen.OrgRole.map(ORG_ROLE_DICT)
-
-    dfSen['SeniorityName'] = dfSen.SeniorityCode.map(SENIORITY_CODE_DICT)
-
-    dfSen = dfSen[["YearCensus", "SWENo", "RoleStartDate", "NewOrNot", "RoleEndDate", "LeftOrNot",
-                   "AgencyWorker", "OrgRole", "OrgRoleName", "SeniorityCode", "SeniorityName"]]
-
+    dfSen["OrgRoleName"] = dfSen.OrgRole.map(
+        {int(key): ORG_ROLE_DICT[key] for key in ORG_ROLE_DICT}
+    )
+    dfSen["SeniorityName"] = dfSen.SeniorityCode.map(SENIORITY_CODE_DICT)
```

Comment:
Hey Stephen, if you could just apply the above .map fix to this row as well that would be great. Also the folder you made with the blank.txt files have disappeared, I think because they are now empty (which I didn't think would completely erase them so my bad) but if you could just add those blank.txt files back in that would fix this